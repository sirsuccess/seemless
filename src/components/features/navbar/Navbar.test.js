import React from "react";
import renderer from "react-test-renderer";

import Component from "./Navbar";

it("renders correctly when called ", () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
