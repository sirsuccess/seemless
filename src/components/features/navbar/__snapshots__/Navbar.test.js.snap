// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<div
  className="Navbar"
>
  <a
    href="http://www.google.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    Gmail
  </a>
  <a
    href="http://www.google.com.ng/imghp"
    rel="noopener noreferrer"
    target="_blank"
  >
    Images
  </a>
  <i
    class="fa fa-th"
  />
  <div
    style={
      Object {
        "backgroundColor": "#4485f2",
        "borderRadius": "3px",
        "color": "#fff",
        "fontWeight": "600",
        "padding": "0.5em 1em",
      }
    }
  >
    Sign in
  </div>
</div>
`;
